.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -35,54 +44,55 @@ body:
     attributes:
       label: CLI version
       description: |
-        Run the command `npx @react-native-community/cli --version` in your terminal and copy the results here
-      value: "npx @react-native-community/cli -v"
+        Run the command `npx @react-native-community/cli -v` in your terminal and copy the result here.
+      placeholder: "npx @react-native-community/cli -v"
     id: cli
   - type: textarea
     validations:
       required: true
     attributes:
       label: Environment
       description: |
-        Run the command `npx @react-native-community/cli info` in your terminal and copy the results here
-      value: "npx @react-native-community/cli info"
+        Run the command `npx @react-native-community/cli info` in your terminal and copy the result here.
+      placeholder: "npx @react-native-community/cli info"
       render: markdown
     id: env
   - type: textarea
     attributes:
       label: Community Modules
       description: Describe what you community modules you have added to your project. Not sure? Look at your app's `package.json` and share what's in the "dependencies" section.
     id: modules
+  - type: dropdown
+    validations:
+      required: true
+    attributes:
+      label: Target React Native Architecture
+      description: Which React Native architecture are you using?
+      multiple: true
+      options:
+        - "New Architecture"
+        - "Old Architecture"
```

Comment:
Do we need a "don't know" option, or to have that not be required? I don't know if there will be people who just shrug and ay "I used the defaults"

Reply:
I think it's something devs will have to have be familiar with, and without it it's the first question we're going to ask. If if it's not answered we're going to immediately bounce it back to folks.

Maybe we ask the template they used in the description, as a hint?